### PR TITLE
Add cli command

### LINKED ISSUE #135: [FEATURE] CLI command
Generate the dashboard using a CLI command:

``` bash
databricks labs lsql deploy-dashboard ./path/to/directory/with/queries
```

### PR DESCRIPTION
**Pull request by JCZuurmond**
Resolves #135 

``` bash
~/github/databrickslabs/lsql feat/add-cli-command ⇡1 *1 ❯ databricks labs lsql create-dashboard --folder tests/integration/dashboards/one_counter/                                                                                             х INT 15:33:57
15:34:01  INFO [databricks.sdk] Using Databricks CLI authentication
15:34:01  INFO [__main__] Creating dashboard ...
15:34:01  WARN [d.l.lsql.dashboards] Parsing tests/integration/dashboards/one_counter/000_counter.md: Invalid expression / Unexpected token. Line 1, Col: 1.
  # Counter

Below you see an example counter widget. Counter widgets in dashboards are used to display
15:34:02  INFO [__main__] Created dashboard: https://adb-REDACTED.2.azuredatabricks.net/sql/dashboardsv3/REDACTED.
REDACTED
```

Screenshot 2024-06-14 at 15 36 27

Created issue for the WARN #159

### CODE REVIEW

File: labs.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
```
[the remark this answers is not in the document]
Comment (nfx) [reply labs.yml:1]: Add --profile labs-azure-account-admin for your testing purposes. Also, you don't have to install the app - just call databricks labs lsql create-dashboard from this folder and it'll detect labs.yml up the tree

File: src/databricks/labs/lsql/cli.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+@lsql.command
+def create_dashboard(w: WorkspaceClient, folder: Path = Path.cwd()):
+    """Create a dashboard from queries"""
+    logger.info("Creating dashboard ...")
+    dashboards = Dashboards(w)
+    lakeview_dashboard = dashboards.create_dashboard(Path(folder))
+    sdk_dashboard = dashboards.deploy_dashboard(lakeview_dashboard)
+    dashboard_url = f"{w.config.host}/sql/dashboardsv3/{sdk_dashboard.dashboard_id}"
+    logger.info(f"Created dashboard: {dashboard_url}.")
```
Comment (nfx) [inline src/databricks/labs/lsql/cli.py:21]: can we do `webbrowser.open(dashboard_url)` as well as adding `--no-open` flag

File: src/databricks/labs/lsql/cli.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+from databricks.labs.lsql.dashboards import Dashboards
+
+logger = get_logger(__name__)
+lsql = App(__file__)
+
+
+@lsql.command
+def create_dashboard(w: WorkspaceClient, folder: Path = Path.cwd()):
+    """Create a dashboard from queries"""
+    logger.info("Creating dashboard ...")
```
Comment (nfx) [inline src/databricks/labs/lsql/cli.py:16]:
```suggestion
    logger.debug("Creating dashboard ...")
```

might be noise...

File: src/databricks/labs/lsql/cli.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    dashboards = Dashboards(w)
+    lakeview_dashboard = dashboards.create_dashboard(Path(folder))
+    sdk_dashboard = dashboards.deploy_dashboard(lakeview_dashboard)
```
Comment (nfx) [inline src/databricks/labs/lsql/cli.py:19]: does it open deployed dashboard or dashboard-in-progress?

File: src/databricks/labs/lsql/cli.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+from databricks.sdk import WorkspaceClient
+
+from databricks.labs.lsql.dashboards import Dashboards
+
+logger = get_logger(__name__)
+lsql = App(__file__)
+
+
+@lsql.command
+def create_dashboard(w: WorkspaceClient, folder: Path = Path.cwd()):
```
Comment (nfx) [inline src/databricks/labs/lsql/cli.py:14]: add a basic unit test

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/lsql/cli.py (hunk 4 above)
Comment (JCZuurmond) [reply src/databricks/labs/lsql/cli.py:25]: it opens the unpublished dashboard

Comment (JCZuurmond) [reply src/databricks/labs/lsql/cli.py:25]: see screenshot above

**Pull request merged**